packages/mattermost-redux/src/selectors/entities/general.ts

```diff
@@ -11,7 +11,7 @@ import {ClientConfig, FeatureFlags} from 'mattermost-redux/types/config';
 import {isMinimumServerVersion} from 'mattermost-redux/utils/helpers';
 
 export function getConfig(state: GlobalState): Partial<ClientConfig> {
-    return state.entities.general.config;
+    return state.entities.general?.config;
```

Comment:
Why add the `?` here? When would `state.entities.general` ever be undefined?

Reply:
@hmhealey during testing. A incomplete store is being mocked and tests crash

Reply:
If it's trying to access the config in that test though, shouldn't the config be defined then? Regardless, I'm only 0/5 on that since I know redux testing is a pain. We really need an equivalent of the web app's `testConfigureStore` to fill in all the blanks for us